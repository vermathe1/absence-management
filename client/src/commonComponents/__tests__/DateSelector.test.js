import React from "react";
import { shallow } from "enzyme";
import renderer from "react-test-renderer";
import { DateSelector } from "../index";

describe("<DateSelector />", () => {
  it("should render correctly", () => {
    const DATE_TO_USE = new Date("2016");
    const tree = renderer
      .create(<DateSelector selected={DATE_TO_USE} onChange={jest.fn()} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
