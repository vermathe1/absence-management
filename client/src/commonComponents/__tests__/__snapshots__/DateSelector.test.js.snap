// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DateSelector /> should render correctly 1`] = `
<div
  className="react-datepicker-wrapper"
>
  <div
    className="react-datepicker__input-container"
  >
    <input
      className=""
      disabled={false}
      onBlur={[Function]}
      onChange={[Function]}
      onClick={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      readOnly={false}
      type="text"
      value="01/01/2016"
    />
  </div>
</div>
`;
